packages/core/src/button/Button.tsx

```diff
@@ -72,15 +89,40 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
           withBaseName(variant),
           {
             [withBaseName("disabled")]: disabled,
+            [withBaseName(`loading-${variant}`)]: loading,
             [withBaseName("active")]: active,
           },
           className
         )}
+        aria-live={loading !== undefined ? "assertive" : undefined}
```

Comment:
I can't remember the outcome but what was decided on the usage of assertive here?